{"time": "2025-05-12 14:02:15,761", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 14:02:15,764", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 116.65617299079895}"}
{"time": "2025-05-12 14:02:59,913", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.0018439292907714844}"}
{"time": "2025-05-12 14:02:59,933", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.016448020935058594}"}
{"time": "2025-05-12 14:03:01,448", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.0053670406341552734}"}
{"time": "2025-05-12 14:04:10,204", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 14:04:10,206", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 60.479397773742676}"}
{"time": "2025-05-12 15:19:40,050", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.010690927505493164}"}
{"time": "2025-05-12 15:19:40,075", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.021544933319091797}"}
{"time": "2025-05-12 15:19:41,587", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.008842945098876953}"}
{"time": "2025-05-12 15:19:58,988", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 15:19:58,996", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 9.377217054367065}"}
{"time": "2025-05-12 15:20:24,608", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.003178834915161133}"}
{"time": "2025-05-12 15:20:24,617", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.006360769271850586}"}
{"time": "2025-05-12 15:20:25,565", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.007920980453491211}"}
{"time": "2025-05-12 15:23:02,757", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 15:23:02,764", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 150.69070506095886}"}
{"time": "2025-05-12 15:25:10,632", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.004018306732177734}"}
{"time": "2025-05-12 15:25:10,647", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.012977838516235352}"}
{"time": "2025-05-12 15:25:12,241", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.01212310791015625}"}
{"time": "2025-05-12 15:28:26,539", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 15:28:26,545", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 188.44912195205688}"}
{"time": "2025-05-12 15:31:04,123", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.0039060115814208984}"}
{"time": "2025-05-12 15:31:04,136", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.009310007095336914}"}
{"time": "2025-05-12 15:31:06,084", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/13", "path": "/services/view/13", "method": "GET", "status_code": 200, "process_time": 0.01430201530456543}"}
{"time": "2025-05-12 15:31:55,636", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-12 15:31:55,639", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 200, "process_time": 15.46481990814209}"}
{"time": "2025-05-12 15:33:06,546", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.00458216667175293}"}
{"time": "2025-05-12 15:33:06,561", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.01175379753112793}"}
{"time": "2025-05-12 15:33:08,295", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.008382081985473633}"}
{"time": "2025-05-12 15:38:43,665", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 15:38:43,673", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 327.54088521003723}"}
{"time": "2025-05-12 15:41:38,783", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.004318952560424805}"}
{"time": "2025-05-12 15:41:38,796", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.01105809211730957}"}
{"time": "2025-05-12 15:41:40,220", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.005921840667724609}"}
{"time": "2025-05-12 15:42:16,958", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-12 15:42:16,965", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 30.173625946044922}"}
{"time": "2025-05-12 15:42:55,131", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.0048370361328125}"}
{"time": "2025-05-12 15:42:55,145", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.011095046997070312}"}
{"time": "2025-05-12 15:42:56,871", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.00482940673828125}"}
{"time": "2025-05-12 15:44:24,578", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-12 15:44:24,581", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 81.13902020454407}"}
{"time": "2025-05-12 16:07:36,283", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.004380702972412109}"}
{"time": "2025-05-12 16:07:36,307", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.021632909774780273}"}
{"time": "2025-05-12 16:07:37,911", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.011047124862670898}"}
{"time": "2025-05-12 16:09:35,984", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 16:09:35,992", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 109.21730875968933}"}
{"time": "2025-05-12 16:11:31,875", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.004055023193359375}"}
{"time": "2025-05-12 16:11:31,885", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.006108999252319336}"}
{"time": "2025-05-12 16:11:33,602", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.009321928024291992}"}
{"time": "2025-05-12 16:15:21,991", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: tuple index out of range "}
{"time": "2025-05-12 16:15:21,998", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 500, "process_time": 222.40203499794006}"}
{"time": "2025-05-12 16:22:48,565", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.0048329830169677734}"}
{"time": "2025-05-12 16:22:48,584", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.01582193374633789}"}
{"time": "2025-05-12 16:22:49,942", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/1", "path": "/services/view/1", "method": "GET", "status_code": 200, "process_time": 0.013269186019897461}"}
{"time": "2025-05-12 16:23:18,352", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-12 16:23:18,355", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 200, "process_time": 20.61370277404785}"}
{"time": "2025-05-15 14:44:57,054", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.0057065486907958984}"}
{"time": "2025-05-15 14:44:57,083", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.026665925979614258}"}
{"time": "2025-05-15 14:44:59,027", "level": "INFO", "message": "{"url": "http://localhost:8000/services/view/12", "path": "/services/view/12", "method": "GET", "status_code": 200, "process_time": 0.011556863784790039}"}
{"time": "2025-05-15 14:45:55,891", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-15 14:45:55,893", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 200, "process_time": 14.678276062011719}"}
{"time": "2025-05-15 14:47:25,620", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-15 14:47:25,628", "level": "INFO", "message": "{"url": "http://localhost:8000/analysis/analyze", "path": "/analysis/analyze", "method": "POST", "status_code": 200, "process_time": 14.85839319229126}"}
{"time": "2025-05-15 14:50:21,265", "level": "INFO", "message": "{"url": "http://localhost:8000/services", "path": "/services", "method": "GET", "status_code": 307, "process_time": 0.00257110595703125}"}
{"time": "2025-05-15 14:50:21,282", "level": "INFO", "message": "{"url": "http://localhost:8000/services/", "path": "/services/", "method": "GET", "status_code": 200, "process_time": 0.013235092163085938}"}
{"time": "2025-05-16 06:53:48,120", "level": "INFO", "message": "{"url": "http://127.0.0.1:8000/docs", "path": "/docs", "method": "GET", "status_code": 200, "process_time": 0.0040950775146484375}"}
{"time": "2025-05-16 06:53:48,738", "level": "INFO", "message": "{"url": "http://127.0.0.1:8000/openapi.json", "path": "/openapi.json", "method": "GET", "status_code": 200, "process_time": 0.08914303779602051}"}
{"time": "2025-05-16 06:54:53,649", "level": "INFO", "message": "{"url": "http://127.0.0.1:8000/redocs", "path": "/redocs", "method": "GET", "status_code": 404, "process_time": 0.0004930496215820312}"}
{"time": "2025-05-16 06:54:53,745", "level": "INFO", "message": "{"url": "http://127.0.0.1:8000/favicon.ico", "path": "/favicon.ico", "method": "GET", "status_code": 404, "process_time": 0.0005707740783691406}"}
{"time": "2025-05-16 06:55:07,899", "level": "INFO", "message": "{"url": "http://127.0.0.1:8000/redoc", "path": "/redoc", "method": "GET", "status_code": 200, "process_time": 0.0004799365997314453}"}
{"time": "2025-05-16 06:55:08,428", "level": "INFO", "message": "{"url": "http://127.0.0.1:8000/openapi.json", "path": "/openapi.json", "method": "GET", "status_code": 200, "process_time": 0.0018842220306396484}"}
{"time": "2025-05-16 07:49:11,554", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-16 03:49:11.521155", "user_id": null, "status_code": 200, "duration": 0.025528907775878906}"}
{"time": "2025-05-16 07:50:37,726", "level": "INFO", "message": "Base analysis endpoint hit."}
{"time": "2025-05-16 07:50:37,731", "level": "INFO", "message": "{"method": "GET", "path": "/analysis/", "timestamp": "2025-05-16 03:50:37.724049", "user_id": null, "status_code": 200, "duration": 0.004990100860595703}"}
{"time": "2025-05-16 07:50:59,868", "level": "INFO", "message": "{"method": "GET", "path": "/analysis/analyze", "timestamp": "2025-05-16 03:50:59.853977", "user_id": null, "status_code": 405, "duration": 0.007024288177490234}"}
{"time": "2025-05-16 07:52:54,633", "level": "INFO", "message": "{"method": "GET", "path": "/analysis/analyfgfgfze", "timestamp": "2025-05-16 03:52:54.627891", "user_id": null, "status_code": 404, "duration": 0.003916025161743164}"}
{"time": "2025-05-16 09:49:52,090", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/login", "timestamp": "2025-05-16 05:49:40.399617", "user_id": null, "status_code": 200, "duration": 30.071784019470215}"}
{"time": "2025-05-16 09:49:54,916", "level": "INFO", "message": "{"method": "POST", "path": "/login", "timestamp": "2025-05-16 05:49:54.908515", "user_id": null, "status_code": 404, "duration": 2.8170089721679688}"}
{"time": "2025-05-16 09:51:01,570", "level": "INFO", "message": "{"method": "POST", "path": "/login", "timestamp": "2025-05-16 05:51:01.552708", "user_id": null, "status_code": 404, "duration": 25.98865509033203}"}
{"time": "2025-05-16 09:51:51,644", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/login", "timestamp": "2025-05-16 05:51:51.639533", "user_id": null, "status_code": 200, "duration": 5.8356711864471436}"}
{"time": "2025-05-16 09:51:52,544", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 05:51:52.541370", "user_id": null, "status_code": 405, "duration": 0.8922970294952393}"}
{"time": "2025-05-16 09:53:10,911", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 05:52:42.893425", "user_id": null, "status_code": 405, "duration": 33.6907217502594}"}
{"time": "2025-05-16 09:54:35,644", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 05:54:35.625856", "user_id": null, "status_code": 405, "duration": 0.00910806655883789}"}
{"time": "2025-05-16 09:54:38,572", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 05:54:38.567523", "user_id": null, "status_code": 405, "duration": 0.001474142074584961}"}
{"time": "2025-05-16 09:55:59,643", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/login", "timestamp": "2025-05-16 05:55:59.625997", "user_id": null, "status_code": 200, "duration": 0.0061359405517578125}"}
{"time": "2025-05-16 09:55:59,649", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 05:55:59.647060", "user_id": null, "status_code": 405, "duration": 0.0009710788726806641}"}
{"time": "2025-05-16 10:00:46,968", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/login", "timestamp": "2025-05-16 06:00:46.958220", "user_id": null, "status_code": 200, "duration": 0.004141092300415039}"}
{"time": "2025-05-16 10:00:46,974", "level": "INFO", "message": "{"method": "POST", "path": "/login", "timestamp": "2025-05-16 06:00:46.972083", "user_id": null, "status_code": 404, "duration": 0.0016510486602783203}"}
{"time": "2025-05-16 10:00:59,369", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 06:00:59.361612", "user_id": null, "status_code": 405, "duration": 0.007472991943359375}"}
{"time": "2025-05-16 10:05:00,299", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/login", "timestamp": "2025-05-16 06:05:00.292678", "user_id": null, "status_code": 200, "duration": 0.0016510486602783203}"}
{"time": "2025-05-16 10:05:00,305", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 06:05:00.303348", "user_id": null, "status_code": 405, "duration": 0.001251220703125}"}
{"time": "2025-05-16 10:05:56,961", "level": "INFO", "message": "{"method": "GET", "path": "/users", "timestamp": "2025-05-16 06:05:56.943741", "user_id": null, "status_code": 307, "duration": 0.011387825012207031}"}
{"time": "2025-05-16 10:06:02,481", "level": "INFO", "message": "{"method": "GET", "path": "/users/", "timestamp": "2025-05-16 06:05:56.966300", "user_id": null, "status_code": 200, "duration": 5.499861001968384}"}
{"time": "2025-05-16 10:06:02,724", "level": "INFO", "message": "{"method": "GET", "path": "/favicon.ico", "timestamp": "2025-05-16 06:06:02.720718", "user_id": null, "status_code": 404, "duration": 0.002262115478515625}"}
{"time": "2025-05-16 10:06:11,195", "level": "INFO", "message": "{"method": "GET", "path": "/users/login", "timestamp": "2025-05-16 06:06:11.181637", "user_id": null, "status_code": 404, "duration": 0.013109922409057617}"}
{"time": "2025-05-16 10:08:45,052", "level": "INFO", "message": "{"method": "GET", "path": "/users/login", "timestamp": "2025-05-16 06:07:23.193223", "user_id": null, "status_code": 404, "duration": 81.85578227043152}"}
{"time": "2025-05-16 10:09:17,978", "level": "INFO", "message": "{"method": "GET", "path": "/users/login/", "timestamp": "2025-05-16 06:09:17.967367", "user_id": null, "status_code": 405, "duration": 0.004235029220581055}"}
{"time": "2025-05-16 20:45:01,119", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-16 16:45:01.021810", "user_id": null, "status_code": 307, "duration": 0.029621362686157227}"}
{"time": "2025-05-16 20:45:01,242", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-16 16:45:01.136093", "user_id": null, "status_code": 200, "duration": 0.09567904472351074}"}
{"time": "2025-05-16 20:45:04,267", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-16 16:45:04.215025", "user_id": null, "status_code": 200, "duration": 0.04175114631652832}"}
{"time": "2025-05-16 20:50:18,420", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/login", "timestamp": "2025-05-16 16:50:18.400318", "user_id": null, "status_code": 200, "duration": 0.011184215545654297}"}
{"time": "2025-05-16 20:50:18,436", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 16:50:18.425642", "user_id": null, "status_code": 405, "duration": 0.004103899002075195}"}
{"time": "2025-05-16 20:51:11,346", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/login", "timestamp": "2025-05-16 16:51:11.336074", "user_id": null, "status_code": 200, "duration": 0.0019927024841308594}"}
{"time": "2025-05-16 20:51:11,363", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 16:51:11.352964", "user_id": null, "status_code": 405, "duration": 0.002801179885864258}"}
{"time": "2025-05-16 20:51:57,115", "level": "INFO", "message": "{"method": "POST", "path": "/users/login", "timestamp": "2025-05-16 16:51:57.105877", "user_id": null, "status_code": 405, "duration": 0.0010213851928710938}"}
{"time": "2025-05-16 20:52:54,566", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/login", "timestamp": "2025-05-16 16:52:54.550168", "user_id": null, "status_code": 200, "duration": 0.001008749008178711}"}
{"time": "2025-05-16 20:52:54,581", "level": "INFO", "message": "{"method": "POST", "path": "/login", "timestamp": "2025-05-16 16:52:54.571762", "user_id": null, "status_code": 404, "duration": 0.0020270347595214844}"}
{"time": "2025-05-16 20:55:14,029", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/login/", "timestamp": "2025-05-16 16:55:14.021492", "user_id": null, "status_code": 200, "duration": 0.0013301372528076172}"}
{"time": "2025-05-16 20:55:14,041", "level": "INFO", "message": "{"method": "POST", "path": "/login/", "timestamp": "2025-05-16 16:55:14.033076", "user_id": null, "status_code": 404, "duration": 0.002560138702392578}"}
{"time": "2025-05-16 20:55:31,897", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/login/", "timestamp": "2025-05-16 16:55:31.886086", "user_id": null, "status_code": 200, "duration": 0.0011010169982910156}"}
{"time": "2025-05-16 20:55:31,912", "level": "INFO", "message": "{"method": "POST", "path": "/login/", "timestamp": "2025-05-16 16:55:31.902225", "user_id": null, "status_code": 404, "duration": 0.002433300018310547}"}
{"time": "2025-05-16 20:56:29,102", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/login/", "timestamp": "2025-05-16 16:56:29.091326", "user_id": null, "status_code": 200, "duration": 0.0010223388671875}"}
{"time": "2025-05-16 20:56:29,147", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 16:56:29.107663", "user_id": null, "status_code": 422, "duration": 0.03163266181945801}"}
{"time": "2025-05-16 20:59:14,138", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 16:59:14.108717", "user_id": null, "status_code": 422, "duration": 0.0058252811431884766}"}
{"time": "2025-05-16 20:59:39,049", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 16:59:39.037291", "user_id": null, "status_code": 422, "duration": 0.0052797794342041016}"}
{"time": "2025-05-16 21:13:02,794", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:12:49.851389", "user_id": null, "status_code": 500, "duration": 12.93286943435669}"}
{"time": "2025-05-16 21:14:34,346", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:14:14.400442", "user_id": null, "status_code": 500, "duration": 19.936798810958862}"}
{"time": "2025-05-16 21:15:49,709", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:14:50.209693", "user_id": null, "status_code": 500, "duration": 59.480931520462036}"}
{"time": "2025-05-16 21:17:23,890", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:16:19.371020", "user_id": null, "status_code": 500, "duration": 64.51172041893005}"}
{"time": "2025-05-16 21:21:58,413", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:20:42.743839", "user_id": null, "status_code": 500, "duration": 75.65706968307495}"}
{"time": "2025-05-16 21:23:00,887", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:22:48.907511", "user_id": null, "status_code": 500, "duration": 11.972060680389404}"}
{"time": "2025-05-16 21:23:23,022", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:23:08.342428", "user_id": null, "status_code": 500, "duration": 14.673574686050415}"}
{"time": "2025-05-16 21:24:05,008", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:23:31.078126", "user_id": null, "status_code": 500, "duration": 33.92470693588257}"}
{"time": "2025-05-16 21:25:02,207", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:24:45.168514", "user_id": null, "status_code": 500, "duration": 17.02732253074646}"}
{"time": "2025-05-16 21:28:14,510", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:27:52.701302", "user_id": null, "status_code": 500, "error_message": "tuple index out of range", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\routers\\users.py\", line 74, in login_for_access_token\n    logging.ERROR(f\"Login error: {e}\")\n                                       \nIndexError: tuple index out of range\n", "duration": 21.78853487968445}"}
{"time": "2025-05-16 21:31:24,980", "level": "INFO", "message": "{"method": "POST", "path": "/users/register", "timestamp": "2025-05-16 17:31:24.923151", "user_id": null, "status_code": 405, "duration": 0.020258665084838867}"}
{"time": "2025-05-16 21:31:40,437", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 17:31:40.382684", "user_id": null, "status_code": 422, "duration": 0.045134544372558594}"}
{"time": "2025-05-16 21:33:08,948", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 17:33:08.936219", "user_id": null, "status_code": 422, "duration": 0.003707408905029297}"}
{"time": "2025-05-16 21:36:01,770", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 17:36:01.756992", "user_id": null, "status_code": 422, "duration": 0.006367921829223633}"}
{"time": "2025-05-16 21:36:28,481", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:36:23.346851", "user_id": null, "status_code": 500, "error_message": "'int' object is not callable", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\routers\\users.py\", line 74, in login_for_access_token\n    logging.ERROR(f\"Login error: {e}\")\nTypeError: 'int' object is not callable\n", "duration": 5.12968373298645}"}
{"time": "2025-05-16 21:37:20,985", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:36:37.133418", "user_id": null, "status_code": 500, "error_message": "'int' object is not callable", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\routers\\users.py\", line 74, in login_for_access_token\n    # logging.ERROR(f\"Login error: {e}\")\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: 'int' object is not callable\n", "duration": 43.84379696846008}"}
{"time": "2025-05-16 21:37:53,747", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:37:29.709546", "user_id": null, "status_code": 500, "error_message": "'int' object is not callable", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\routers\\users.py\", line 74, in login_for_access_token\n    # logging.ERROR(f\"Login error: {e}\")\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: 'int' object is not callable\n", "duration": 24.028635263442993}"}
{"time": "2025-05-16 21:38:53,113", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:38:06.135579", "user_id": null, "status_code": 500, "duration": 46.97205448150635}"}
{"time": "2025-05-16 21:41:21,147", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:41:18.796987", "user_id": null, "status_code": 500, "duration": 2.340833902359009}"}
{"time": "2025-05-16 21:41:46,757", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 17:41:31.112168", "user_id": null, "status_code": 200, "duration": 15.627235651016235}"}
{"time": "2025-05-16 21:42:45,677", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-16 17:42:45.662253", "user_id": null, "status_code": 307, "duration": 0.0}"}
{"time": "2025-05-16 21:42:45,709", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-16 17:42:45.681975", "user_id": null, "status_code": 200, "duration": 0.01305842399597168}"}
{"time": "2025-05-16 21:42:47,922", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-16 17:42:47.905316", "user_id": null, "status_code": 200, "duration": 0.010639667510986328}"}
{"time": "2025-05-16 21:42:58,082", "level": "ERROR", "message": "HTTPException: 500: Error extracting text from PDF: No /Root object! - Is this really a PDF? "}
{"time": "2025-05-16 21:42:58,091", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-16 17:42:57.874480", "user_id": null, "status_code": 500, "duration": 0.2082064151763916}"}
{"time": "2025-05-16 21:48:19,290", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-16 17:48:19.274624", "user_id": null, "status_code": 307, "duration": 0.010494470596313477}"}
{"time": "2025-05-16 21:48:19,312", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-16 17:48:19.290695", "user_id": null, "status_code": 200, "duration": 0.01580977439880371}"}
{"time": "2025-05-16 21:48:21,869", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-16 17:48:21.855799", "user_id": null, "status_code": 200, "duration": 0.007091045379638672}"}
{"time": "2025-05-16 21:49:01,146", "level": "ERROR", "message": "HTTPException: 500: Error extracting text from PDF: No /Root object! - Is this really a PDF? "}
{"time": "2025-05-16 21:49:35,751", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-16 17:48:29.391262", "user_id": null, "status_code": 500, "duration": 66.35703825950623}"}
{"time": "2025-05-16 22:09:04,476", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/register/", "timestamp": "2025-05-16 18:09:04.456522", "user_id": null, "status_code": 200, "duration": 0.007620811462402344}"}
{"time": "2025-05-16 22:09:04,522", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:09:04.485863", "user_id": null, "status_code": 422, "duration": 0.03214621543884277}"}
{"time": "2025-05-16 22:10:21,427", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:10:21.409492", "user_id": null, "status_code": 422, "duration": 0.00852823257446289}"}
{"time": "2025-05-16 22:10:26,695", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:10:26.674683", "user_id": null, "status_code": 422, "duration": 0.006796836853027344}"}
{"time": "2025-05-16 22:12:16,323", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:12:16.291819", "user_id": null, "status_code": 422, "duration": 0.018508434295654297}"}
{"time": "2025-05-16 22:15:47,693", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:15:47.671052", "user_id": null, "status_code": 422, "duration": 0.011232137680053711}"}
{"time": "2025-05-16 22:19:39,320", "level": "INFO", "message": "{"method": "OPTIONS", "path": "/users/register/", "timestamp": "2025-05-16 18:19:39.307620", "user_id": null, "status_code": 200, "duration": 0.0}"}
{"time": "2025-05-16 22:20:02,421", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:19:39.324069", "user_id": null, "status_code": 500, "error_message": "'UserCreate' object has no attribute 'name'", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\routers\\users.py\", line 44, in create_user\n    db_user = SQLUser(id=str(uuid.uuid4()), name=user.name, email=user.email, password=hashed_password)\n                                                 ^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\pydantic\\main.py\", line 994, in __getattr__\n    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')\nAttributeError: 'UserCreate' object has no attribute 'name'\n", "duration": 23.09239625930786}"}
{"time": "2025-05-16 22:21:28,613", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:20:58.193765", "user_id": null, "status_code": 500, "error_message": "'UserCreate' object has no attribute 'name'", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 214, in run_endpoint_function\n    return await run_in_threadpool(dependant.call, **values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\concurrency.py\", line 37, in run_in_threadpool\n    return await anyio.to_thread.run_sync(func)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\to_thread.py\", line 56, in run_sync\n    return await get_async_backend().run_sync_in_worker_thread(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 2470, in run_sync_in_worker_thread\n    return await future\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\_backends\\_asyncio.py\", line 967, in run\n    result = context.run(func, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\routers\\users.py\", line 44, in create_user\n    db_user = SQLUser(id=str(uuid.uuid4()), username=user.username, email=user.email, password=hashed_password)\n                                                 ^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\pydantic\\main.py\", line 994, in __getattr__\n    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')\nAttributeError: 'UserCreate' object has no attribute 'name'\n", "duration": 30.399041891098022}"}
{"time": "2025-05-16 22:23:34,408", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:22:51.268480", "user_id": null, "status_code": 500, "error_message": "2 validation errors:\n  {'type': 'string_type', 'loc': ('response', 'role'), 'msg': 'Input should be a valid string', 'input': None}\n  {'type': 'string_type', 'loc': ('response', 'avatar'), 'msg': 'Input should be a valid string', 'input': None}\n", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 148, in call_next\n    message = await recv_stream.receive()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\anyio\\streams\\memory.py\", line 126, in receive\n    raise EndOfStream from None\nanyio.EndOfStream\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\main.py\", line 56, in log_requests\n    response = await call_next(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 156, in call_next\n    raise app_exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\base.py\", line 141, in coro\n    await self.app(scope, receive_or_disconnect, send_no_error)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 327, in app\n    content = await serialize_response(\n              ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\hatem\\Desktop\\Hatem\\Eight-Company\\projects\\tahlyl-backend\\venv\\Lib\\site-packages\\fastapi\\routing.py\", line 176, in serialize_response\n    raise ResponseValidationError(\nfastapi.exceptions.ResponseValidationError: 2 validation errors:\n  {'type': 'string_type', 'loc': ('response', 'role'), 'msg': 'Input should be a valid string', 'input': None}\n  {'type': 'string_type', 'loc': ('response', 'avatar'), 'msg': 'Input should be a valid string', 'input': None}\n\n", "duration": 43.135485887527466}"}
{"time": "2025-05-16 22:26:05,153", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:25:42.165206", "user_id": null, "status_code": 400, "duration": 22.982361316680908}"}
{"time": "2025-05-16 22:27:38,471", "level": "INFO", "message": "{"method": "POST", "path": "/users/register/", "timestamp": "2025-05-16 18:27:29.984574", "user_id": null, "status_code": 201, "duration": 8.47626256942749}"}
{"time": "2025-05-16 22:28:20,042", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-16 18:28:13.843663", "user_id": null, "status_code": 200, "duration": 6.183244943618774}"}
{"time": "2025-05-21 15:13:15,065", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-21 11:13:15.048862", "user_id": null, "status_code": 307, "duration": 0.0052340030670166016}"}
{"time": "2025-05-21 15:13:15,087", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-21 11:13:15.070800", "user_id": null, "status_code": 200, "duration": 0.01552891731262207}"}
{"time": "2025-05-21 15:13:39,256", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-21 11:13:31.869403", "user_id": null, "status_code": 500, "duration": 7.385365962982178}"}
{"time": "2025-05-21 15:15:22,546", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-21 11:14:39.841353", "user_id": null, "status_code": 500, "duration": 42.70276403427124}"}
{"time": "2025-05-21 15:15:52,917", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-21 11:15:38.334605", "user_id": null, "status_code": 500, "duration": 14.579794883728027}"}
{"time": "2025-05-21 15:16:12,546", "level": "INFO", "message": "{"method": "POST", "path": "/users/login/", "timestamp": "2025-05-21 11:16:09.734503", "user_id": null, "status_code": 200, "duration": 2.811126947402954}"}
{"time": "2025-05-21 15:16:20,471", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-21 11:16:20.466384", "user_id": null, "status_code": 307, "duration": 0.0017430782318115234}"}
{"time": "2025-05-21 15:16:20,481", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-21 11:16:20.475393", "user_id": null, "status_code": 200, "duration": 0.004038095474243164}"}
{"time": "2025-05-22 05:34:32,366", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:34:32.349853', 'GET', '/services', 307, 'Temporary Redirect', 0.011389970779418945, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:34:32,367", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 01:34:32.349853", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.011389970779418945}"}
{"time": "2025-05-22 05:34:32,384", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:34:32.374653', 'GET', '/services/', 200, 'OK', 0.009447813034057617, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:34:32,384", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 01:34:32.374653", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.009447813034057617}"}
{"time": "2025-05-22 05:41:52,607", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:41:52.598213', 'GET', '/services', 307, 'Temporary Redirect', 0.0030570030212402344, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:41:52,608", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 01:41:52.598213", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0030570030212402344}"}
{"time": "2025-05-22 05:41:52,701", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:41:52.690560', 'GET', '/services/', 200, 'OK', 0.01052713394165039, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:41:52,701", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 01:41:52.690560", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.01052713394165039}"}
{"time": "2025-05-22 05:44:58,681", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:44:58.678361', 'GET', '/services', 307, 'Temporary Redirect', 0.0015697479248046875, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:44:58,682", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 01:44:58.678361", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0015697479248046875}"}
{"time": "2025-05-22 05:44:58,687", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:44:58.684120', 'GET', '/services/', 200, 'OK', 0.002930164337158203, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:44:58,687", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 01:44:58.684120", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002930164337158203}"}
{"time": "2025-05-22 05:45:00,677", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:45:00.674817', 'GET', '/services', 307, 'Temporary Redirect', 0.002129793167114258, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:45:00,677", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 01:45:00.674817", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.002129793167114258}"}
{"time": "2025-05-22 05:45:00,681", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 01:45:00.679158', 'GET', '/services/', 200, 'OK', 0.0017840862274169922, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 05:45:00,681", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 01:45:00.679158", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0017840862274169922}"}
{"time": "2025-05-22 07:34:13,846", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:34:13.838679', 'GET', '/services', 307, 'Temporary Redirect', 0.0014522075653076172, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:34:13,848", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:34:13.838679", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0014522075653076172}"}
{"time": "2025-05-22 07:34:13,861", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:34:13.850988', 'GET', '/services/', 200, 'OK', 0.010250091552734375, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:34:13,861", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:34:13.850988", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.010250091552734375}"}
{"time": "2025-05-22 07:36:16,842", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:36:16.836595', 'GET', '/services', 307, 'Temporary Redirect', 0.0020880699157714844, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:36:16,844", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:36:16.836595", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0020880699157714844}"}
{"time": "2025-05-22 07:36:16,852", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:36:16.847933', 'GET', '/services/', 200, 'OK', 0.004266977310180664, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:36:16,853", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:36:16.847933", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004266977310180664}"}
{"time": "2025-05-22 07:36:20,823", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:36:20.822023', 'GET', '/services', 307, 'Temporary Redirect', 0.0009648799896240234, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:36:20,824", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:36:20.822023", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0009648799896240234}"}
{"time": "2025-05-22 07:36:20,827", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:36:20.825534', 'GET', '/services/', 200, 'OK', 0.0018322467803955078, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:36:20,827", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:36:20.825534", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0018322467803955078}"}
{"time": "2025-05-22 07:43:04,739", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:43:04.728227', 'GET', '/services', 307, 'Temporary Redirect', 0.0025179386138916016, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:43:04,750", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:43:04.728227", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0025179386138916016}"}
{"time": "2025-05-22 07:43:04,778", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:43:04.752212', 'GET', '/services/', 200, 'OK', 0.02579021453857422, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:43:04,778", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:43:04.752212", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.02579021453857422}"}
{"time": "2025-05-22 07:45:39,371", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:45:39.358518', 'GET', '/services', 307, 'Temporary Redirect', 0.0023889541625976562, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:45:39,372", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:45:39.358518", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0023889541625976562}"}
{"time": "2025-05-22 07:45:39,388", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:45:39.383603', 'GET', '/services/', 200, 'OK', 0.004215717315673828, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:45:39,388", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:45:39.383603", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004215717315673828}"}
{"time": "2025-05-22 07:48:41,298", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:48:41.295292', 'GET', '/services', 307, 'Temporary Redirect', 0.001271963119506836, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:48:41,302", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:48:41.295292", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.001271963119506836}"}
{"time": "2025-05-22 07:48:41,312", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:48:41.307909', 'GET', '/services/', 200, 'OK', 0.00400996208190918, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:48:41,312", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:48:41.307909", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.00400996208190918}"}
{"time": "2025-05-22 07:49:50,805", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:49:50.798678', 'GET', '/services', 307, 'Temporary Redirect', 0.0013079643249511719, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:49:50,806", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 03:49:50.798678", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0013079643249511719}"}
{"time": "2025-05-22 07:49:50,840", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:49:50.821525', 'GET', '/services/', 200, 'OK', 0.018735170364379883, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:49:50,841", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 03:49:50.821525", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.018735170364379883}"}
{"time": "2025-05-22 07:50:52,668", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:50:52.637175', 'GET', '/services/view/1', 200, 'OK', 0.031013965606689453, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:50:52,668", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:50:52.637175", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.031013965606689453}"}
{"time": "2025-05-22 07:53:57,915", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:57.898507', 'GET', '/services/view/1', 200, 'OK', 0.015233039855957031, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:57,915", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:57.898507", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.015233039855957031}"}
{"time": "2025-05-22 07:53:58,755", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:58.752764', 'GET', '/services/view/1', 200, 'OK', 0.00238800048828125, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:58,755", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:58.752764", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.00238800048828125}"}
{"time": "2025-05-22 07:53:58,878", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:58.870170', 'GET', '/services/view/1', 200, 'OK', 0.006853818893432617, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:58,879", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:58.870170", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.006853818893432617}"}
{"time": "2025-05-22 07:53:59,052", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:59.050107', 'GET', '/services/view/1', 200, 'OK', 0.0022940635681152344, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:59,053", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:59.050107", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0022940635681152344}"}
{"time": "2025-05-22 07:53:59,219", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:59.216077', 'GET', '/services/view/1', 200, 'OK', 0.0022318363189697266, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:59,219", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:59.216077", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0022318363189697266}"}
{"time": "2025-05-22 07:53:59,408", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:59.405623', 'GET', '/services/view/1', 200, 'OK', 0.002391815185546875, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:59,408", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:59.405623", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002391815185546875}"}
{"time": "2025-05-22 07:53:59,576", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:59.573796', 'GET', '/services/view/1', 200, 'OK', 0.0021393299102783203, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:59,576", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:59.573796", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0021393299102783203}"}
{"time": "2025-05-22 07:53:59,763", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:59.755050', 'GET', '/services/view/1', 200, 'OK', 0.0073699951171875, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:59,764", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:59.755050", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0073699951171875}"}
{"time": "2025-05-22 07:53:59,934", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:53:59.928766', 'GET', '/services/view/1', 200, 'OK', 0.0051577091217041016, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:53:59,934", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:53:59.928766", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0051577091217041016}"}
{"time": "2025-05-22 07:54:00,104", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:00.101286', 'GET', '/services/view/1', 200, 'OK', 0.002753019332885742, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:00,104", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:00.101286", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002753019332885742}"}
{"time": "2025-05-22 07:54:00,287", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:00.285065', 'GET', '/services/view/1', 200, 'OK', 0.0025680065155029297, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:00,288", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:00.285065", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0025680065155029297}"}
{"time": "2025-05-22 07:54:00,488", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:00.484193', 'GET', '/services/view/1', 200, 'OK', 0.004111289978027344, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:00,488", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:00.484193", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004111289978027344}"}
{"time": "2025-05-22 07:54:00,657", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:00.654786', 'GET', '/services/view/1', 200, 'OK', 0.0021963119506835938, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:00,657", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:00.654786", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0021963119506835938}"}
{"time": "2025-05-22 07:54:00,831", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:00.829086', 'GET', '/services/view/1', 200, 'OK', 0.0022590160369873047, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:00,831", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:00.829086", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0022590160369873047}"}
{"time": "2025-05-22 07:54:01,015", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:01.012665', 'GET', '/services/view/1', 200, 'OK', 0.0027968883514404297, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:01,016", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:01.012665", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0027968883514404297}"}
{"time": "2025-05-22 07:54:01,200", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:01.197315', 'GET', '/services/view/1', 200, 'OK', 0.0022339820861816406, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:01,200", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:01.197315", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0022339820861816406}"}
{"time": "2025-05-22 07:54:01,383", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:01.380691', 'GET', '/services/view/1', 200, 'OK', 0.0022640228271484375, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:01,383", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:01.380691", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0022640228271484375}"}
{"time": "2025-05-22 07:54:01,566", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:01.563625', 'GET', '/services/view/1', 200, 'OK', 0.0024569034576416016, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:01,566", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:01.563625", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0024569034576416016}"}
{"time": "2025-05-22 07:54:01,742", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:01.737593', 'GET', '/services/view/1', 200, 'OK', 0.004950761795043945, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:01,743", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:01.737593", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004950761795043945}"}
{"time": "2025-05-22 07:54:01,926", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:01.923523', 'GET', '/services/view/1', 200, 'OK', 0.002496004104614258, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:01,926", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:01.923523", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002496004104614258}"}
{"time": "2025-05-22 07:54:02,112", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:02.109915', 'GET', '/services/view/1', 200, 'OK', 0.0024030208587646484, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:02,112", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:02.109915", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0024030208587646484}"}
{"time": "2025-05-22 07:54:02,284", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:02.281914', 'GET', '/services/view/1', 200, 'OK', 0.002053976058959961, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:02,284", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:02.281914", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002053976058959961}"}
{"time": "2025-05-22 07:54:02,467", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:02.464975', 'GET', '/services/view/1', 200, 'OK', 0.002579212188720703, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:02,468", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:02.464975", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002579212188720703}"}
{"time": "2025-05-22 07:54:02,647", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:02.643122', 'GET', '/services/view/1', 200, 'OK', 0.004138946533203125, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:02,647", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:02.643122", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004138946533203125}"}
{"time": "2025-05-22 07:54:02,839", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:54:02.836364', 'GET', '/services/view/1', 200, 'OK', 0.002415180206298828, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:54:02,839", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:54:02.836364", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.002415180206298828}"}
{"time": "2025-05-22 07:55:45,201", "level": "ERROR", "message": "Failed to save log to DB: (sqlite3.OperationalError) table api_logs has no column named status_description
[SQL: INSERT INTO api_logs (timestamp, method, path, status_code, status_description, duration, user_id, error_message, traceback) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('2025-05-22 03:55:45.180366', 'GET', '/services/view/1', 200, 'OK', 0.018884897232055664, None, None, None)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)"}
{"time": "2025-05-22 07:55:45,203", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 03:55:45.180366", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.018884897232055664}"}
{"time": "2025-05-22 08:04:48,460", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 04:04:48.389472", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.06432104110717773}"}
{"time": "2025-05-22 08:36:09,702", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 04:36:09.672352", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.011237859725952148}"}
{"time": "2025-05-22 08:36:09,718", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 04:36:09.705564", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.012233972549438477}"}
{"time": "2025-05-22 08:48:46,501", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 04:48:46.481910", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.00574946403503418}"}
{"time": "2025-05-22 08:48:46,518", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 04:48:46.504544", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.013158798217773438}"}
{"time": "2025-05-22 08:48:48,554", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 04:48:48.547358", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.006394863128662109}"}
{"time": "2025-05-22 08:49:25,195", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 08:49:25,198", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 04:49:14.963320", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 10.234421968460083}"}
{"time": "2025-05-22 09:53:06,837", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 05:53:06.820182", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.006373167037963867}"}
{"time": "2025-05-22 09:53:06,851", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 05:53:06.839584", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.011003255844116211}"}
{"time": "2025-05-22 09:53:09,025", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 05:53:09.017853", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0060307979583740234}"}
{"time": "2025-05-22 09:54:34,626", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: 'NoneType' object has no attribute 'report_type' "}
{"time": "2025-05-22 09:54:34,632", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 05:53:15.877372", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 78.75286388397217}"}
{"time": "2025-05-22 09:56:12,594", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 05:56:12.469521", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.10965609550476074}"}
{"time": "2025-05-22 10:00:10,743", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: 'NoneType' object has no attribute 'report_type' "}
{"time": "2025-05-22 10:00:10,767", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 05:56:27.942678", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 222.8215069770813}"}
{"time": "2025-05-22 10:03:53,254", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:03:53.207304", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.024631023406982422}"}
{"time": "2025-05-22 10:03:53,273", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:03:53.259272", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.013562917709350586}"}
{"time": "2025-05-22 10:03:54,761", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:03:54.750666", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.009770870208740234}"}
{"time": "2025-05-22 10:04:29,266", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:04:29,269", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:04:03.196349", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 26.072633028030396}"}
{"time": "2025-05-22 10:05:52,442", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:05:52,446", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:05:19.631750", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 32.8127281665802}"}
{"time": "2025-05-22 10:10:38,568", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:10:38.532124", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.006394863128662109}"}
{"time": "2025-05-22 10:10:38,593", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:10:38.571328", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.02068305015563965}"}
{"time": "2025-05-22 10:10:39,715", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:10:39.706512", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.007866859436035156}"}
{"time": "2025-05-22 10:11:10,390", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:11:10,393", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:10:48.222859", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 22.169183015823364}"}
{"time": "2025-05-22 10:12:13,078", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:12:13,082", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:11:34.248243", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 38.833070278167725}"}
{"time": "2025-05-22 10:21:13,106", "level": "ERROR", "message": "HTTPException: 500: Error processing report analysis: cannot access local variable 'db_result' where it is not associated with a value "}
{"time": "2025-05-22 10:21:13,116", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:20:33.340348", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 39.771766901016235}"}
{"time": "2025-05-22 10:23:38,500", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:23:38.480497", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.009902238845825195}"}
{"time": "2025-05-22 10:23:38,540", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:23:38.508569", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.030064105987548828}"}
{"time": "2025-05-22 10:23:40,075", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:23:40.061764", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.011705160140991211}"}
{"time": "2025-05-22 10:24:35,554", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:24:35,558", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:23:47.147497", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 48.409595012664795}"}
{"time": "2025-05-22 10:25:54,318", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 10:25:54,322", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:25:11.079421", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 43.242839097976685}"}
{"time": "2025-05-22 10:26:52,197", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:26:52.180364", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0014922618865966797}"}
{"time": "2025-05-22 10:26:52,209", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:26:52.199340", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0071599483489990234}"}
{"time": "2025-05-22 10:26:53,883", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:26:53.870872", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.009092092514038086}"}
{"time": "2025-05-22 10:27:07,028", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:27:07,031", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:27:01.463885", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 5.566091060638428}"}
{"time": "2025-05-22 10:27:52,696", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 10:27:52,699", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:27:47.549659", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 5.149389982223511}"}
{"time": "2025-05-22 10:29:49,207", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:29:49.197383", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.005563020706176758}"}
{"time": "2025-05-22 10:29:49,219", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:29:49.211903", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0039103031158447266}"}
{"time": "2025-05-22 10:32:06,320", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:32:06.270127", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.04742884635925293}"}
{"time": "2025-05-22 10:33:57,204", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:34:00,395", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:32:21.503506", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 98.88249111175537}"}
{"time": "2025-05-22 10:37:53,350", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 10:37:53,362", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:35:25.729509", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 147.62737011909485}"}
{"time": "2025-05-22 10:38:54,835", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:38:54.817680", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0025458335876464844}"}
{"time": "2025-05-22 10:38:54,847", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:38:54.838440", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.007888078689575195}"}
{"time": "2025-05-22 10:38:56,272", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:38:56.259125", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.010722875595092773}"}
{"time": "2025-05-22 10:39:24,914", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 10:39:24,926", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:39:12.920643", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 12.003641128540039}"}
{"time": "2025-05-22 10:39:29,025", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:39:29.020858", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.0037000179290771484}"}
{"time": "2025-05-22 10:41:07,969", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 10:41:07,982", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:39:37.471393", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 90.50620794296265}"}
{"time": "2025-05-22 10:42:29,298", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:42:29.286088", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.007477760314941406}"}
{"time": "2025-05-22 10:42:29,312", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:42:29.300846", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.010223865509033203}"}
{"time": "2025-05-22 10:42:31,688", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:42:31.675527", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.010608196258544922}"}
{"time": "2025-05-22 10:43:09,744", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 10:43:09,758", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:42:52.690839", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 17.06135082244873}"}
{"time": "2025-05-22 10:43:29,600", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 06:43:29.585014", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.006571054458618164}"}
{"time": "2025-05-22 10:43:29,608", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 06:43:29.602528", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004665851593017578}"}
{"time": "2025-05-22 10:43:31,107", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 06:43:31.089295", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.014996051788330078}"}
{"time": "2025-05-22 10:43:56,663", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:43:56,670", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:43:37.359936", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 19.30800485610962}"}
{"time": "2025-05-22 10:50:09,949", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:50:14,370", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:44:53.905265", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 320.4564459323883}"}
{"time": "2025-05-22 10:50:49,140", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 10:50:49,145", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 06:50:39.989204", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 9.155067205429077}"}
{"time": "2025-05-22 11:11:05,701", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 07:11:05.649511", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.009196043014526367}"}
{"time": "2025-05-22 11:11:05,733", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 07:11:05.707222", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.024626970291137695}"}
{"time": "2025-05-22 11:11:07,267", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 07:11:07.256486", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.009322643280029297}"}
{"time": "2025-05-22 11:11:23,962", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 11:11:23,964", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 07:11:16.087357", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 7.876343011856079}"}
{"time": "2025-05-22 11:13:26,261", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 07:13:26.247470", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.009409904479980469}"}
{"time": "2025-05-22 11:13:26,276", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 07:13:26.264654", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.010727167129516602}"}
{"time": "2025-05-22 11:13:27,874", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 07:13:27.860927", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.011316061019897461}"}
{"time": "2025-05-22 11:15:47,428", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 11:15:47,447", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 07:13:37.039267", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 130.40255188941956}"}
{"time": "2025-05-22 11:16:04,954", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 07:16:04.940574", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.01063394546508789}"}
{"time": "2025-05-22 11:16:04,962", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 07:16:04.956659", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.004601001739501953}"}
{"time": "2025-05-22 11:16:06,032", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 07:16:06.017916", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.012040138244628906}"}
{"time": "2025-05-22 11:17:55,815", "level": "ERROR", "message": "Error processing analysis: tuple index out of range"}
{"time": "2025-05-22 11:17:55,834", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 07:16:12.368415", "user_id": null, "status_code": 500, "status_description": "Internal Server Error", "duration": 103.46222496032715}"}
{"time": "2025-05-22 11:19:08,905", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 07:19:08.862829", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.008565902709960938}"}
{"time": "2025-05-22 11:19:08,928", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 07:19:08.908341", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.01778101921081543}"}
{"time": "2025-05-22 11:19:10,285", "level": "INFO", "message": "{"method": "GET", "path": "/services/view/1", "timestamp": "2025-05-22 07:19:10.266934", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.015670061111450195}"}
{"time": "2025-05-22 11:20:30,492", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 11:20:30,499", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 07:19:18.465748", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 72.03133296966553}"}
{"time": "2025-05-22 11:24:57,035", "level": "INFO", "message": "Email sent successfully to hatem.gadallah@gmail.com."}
{"time": "2025-05-22 11:24:57,049", "level": "INFO", "message": "{"method": "POST", "path": "/analysis/analyze", "timestamp": "2025-05-22 07:24:40.299811", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 16.740859985351562}"}
{"time": "2025-05-22 11:27:15,039", "level": "INFO", "message": "{"method": "GET", "path": "/services", "timestamp": "2025-05-22 07:27:15.032413", "user_id": null, "status_code": 307, "status_description": "Temporary Redirect", "duration": 0.0026328563690185547}"}
{"time": "2025-05-22 11:27:15,051", "level": "INFO", "message": "{"method": "GET", "path": "/services/", "timestamp": "2025-05-22 07:27:15.043739", "user_id": null, "status_code": 200, "status_description": "OK", "duration": 0.006990909576416016}"}
